eyeball-im/src/vector.rs

```diff
@@ -46,7 +46,7 @@ impl<T: Clone + 'static> ObservableVector<T> {
     ///
     /// # Panics
     ///
-    /// Panics if the capacity is `0`, or larger than `usize::MAX / 2`.
+    /// Panics if the capacity is `0`, or overflows.
```

Comment:
This panic message was copied from https://docs.rs/tokio/latest/tokio/sync/broadcast/fn.channel.html, and I think we should not change.

Reply:
Okay, the code is here, https://github.com/tokio-rs/tokio/blob/17d8c2b29d94550f504d8fd76d8d8aaf66095864/tokio/src/sync/broadcast.rs#L543-L546, so it's not larger than, but larger or equal to.